packages/fx-core/src/plugins/resource/simpleauth/index.ts

```diff
@@ -24,11 +24,7 @@ export class SimpleAuthPlugin implements Plugin {
   name = "fx-resource-simple-auth";
   displayName = "Simple Auth";
   activate(solutionSettings: AzureSolutionSettings): boolean {
-    if (solutionSettings?.migrateFromV1 || isVsCallingCli()) {
-      return false;
-    }
-    const cap = solutionSettings.capabilities || [];
-    return solutionSettings.hostType === HostTypeOptionAzure.id && cap.includes(TabOptionItem.id);
+    return false;
```

Comment:
if we return false here, can simple auth plugin enabled for old projects?

Reply:
Simple auth plugin can get template version from fx-core/src/common/template-config.json. Maybe we can decide whether to disable it according to template version?

Reply:
Here we will decide with the active plugins in project settings.